packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1563,6 +1563,11 @@ class EditableText extends StatefulWidget {
   /// Called for each tap up that occurs outside of the [TextFieldTapRegion]
   /// group when the text field is focused.
   ///
+  /// If this is null, [EditableTextTapUpOutsideIntent] will be invoked. In the
+  /// default implementation, this is a no-op. To change this behavior, a
+  /// callback may be set here or [EditableTextTapUpOutsideIntent] may be
+  /// overridden.
```

Comment:
Really small documentation nit: It might be easier to read this in active vs. passive voice. So instead:

```suggestion
  /// default implementation, this is a no-op. To change this behavior, set a
  /// callback here or override [EditableTextTapUpOutsideIntent].
```

or something similar.